App.js

```diff
@@ -15,6 +16,17 @@ import { RestaurantsContextProvider } from "./src/services/restaurants/restauran
 import { LocationContextProvider } from "./src/services/location/location.context";
 import { FavouritesContextProvider } from "./src/services/favourites/favourites.context";
 
+const firebaseConfig = {
+  apiKey: "AIzaSyBmKwzXIQ_7iufDh4U6GyU_4Wc-hyDEnf8",
+  authDomain: "mealstogo-b2612.firebaseapp.com",
+  projectId: "mealstogo-b2612",
+  storageBucket: "mealstogo-b2612.appspot.com",
+  messagingSenderId: "158582890553",
+  appId: "1:158582890553:web:2ef50439fcd80f2ede2de8",
+};
+
+firebase.initializeApp(firebaseConfig);
```

Comment:
If you are on firebase 9 please checkout master for a more up to date reference:
https://github.com/mobinni/MealsToGo/blob/master/App.js

Reply:
I'm following your course to stay up to date with React Native and it's super awesome so far!!

For any users that don't want to push these sensitive firebase config variables to their GitHub repos (ex. `apiKey`, `appId`, etc.), they could use `react-native-dotenv` to easily store these config variables locally on their computers without having them committed to their repo.

Steps to achieve this:

1.) run `yarn add react-native-dotenv`

2.) Create a `.env` file in the root directory of your project and add your environment variables:
````
API_KEY="<your_api_key>"
AUTH_DOMAIN="<your_auth_domain>"
PROJECT_ID="<your_project_id>"
STORAGE_BUCKET="<your_storage_bucket>"
MESSAGING_SENDER_ID="<your_messaging_sender_id>"
APP_ID="<your_app_id>"
````

3.) Edit your `babel.config.js` file to include the `react-native-dotenv` plugin:
````
module.exports = function(api) {
  api.cache(true);
  return {
    presets: ['babel-preset-expo'],
    plugins: [
      ['module:react-native-dotenv', {
        moduleName: '@env',
        path: '.env',
      }],
    ],
  };
};
````

4.) Import and use the environment variables in your `App.js` file:
````
import { initializeApp } from "firebase/app";
import {
  API_KEY,
  AUTH_DOMAIN,
  PROJECT_ID,
  STORAGE_BUCKET,
  MESSAGING_SENDER_ID,
  APP_ID,
} from "@env";

const firebaseConfig = {
  apiKey: API_KEY,
  authDomain: AUTH_DOMAIN,
  projectId: PROJECT_ID,
  storageBucket: STORAGE_BUCKET,
  messagingSenderId: MESSAGING_SENDER_ID,
  appId: APP_ID,
};

initializeApp(firebaseConfig);
````

5.) Make sure your `.env` file is included in your `.gitignore` file to prevent it from being committed to your repository. Add the following line to your `.gitignore` file:
````
.env
````

6.) Keep `.env` file safe and secure and only share with collaborators and use secrets management for your CI/CD pipelines.